spark-connector/spark-common/build.gradle.kts

```diff
@@ -123,6 +130,9 @@ dependencies {
     exclude("org.glassfish.jersey.inject")
   }
   testImplementation("org.scala-lang.modules:scala-collection-compat_$scalaVersion:$scalaCollectionCompatVersion")
+  testImplementation("org.apache.spark:spark-catalyst_$scalaVersion:$sparkVersion")
```

Comment:
why add these dependencies?

Reply:
Now the paimon spark runtime jar is incompatible  with scala2.13 , because it is built with scala2.12 in spark3.3 , spark3.4 and spark3.5 module